Automattic/jetpack
# CLI: Add "Write" to Jetpack Changelog

#### Changes proposed in this Pull Request:

* Adds the `write` changelogger command to `jetpack changelog`

#### Jetpack product discussion


This takes a similar approach to @samiff's much simpler approach when creating the [Docker CLI](https://github.com/Automattic/jetpack/pull/19366), so if this works out, we could likely refactor the rest of the commands to work similarly. 
#### Does this pull request change what data or activity we track or use?


n/a
#### Testing instructions:






The "write" command removes all of the changelog files and adds their contents to the project's `CHANGELOG.md` file. This is destructive to the changelog files, so just don't commit any testing changes :) 

* Make sure a project has changelog files. (under projects/[project]/changelog)
* Use `jetpack changelog write` and follow the prompts to select a project.
* Upon success, check the project's CHANGELOG.md file and make sure it added the changelog file contents and removed the actual changelog files from the changelog folder.  
* It accepts passing a project directly, ie `jetpack changelog write plugins/jetpack`

It also accepts a lot of passthrough arguments, but truthfully I wasn't too sure how to test some of them ([--use-version USE-VERSION] [--use-significance USE-SIGNIFICANCE]), so any help with testing that would be cool. The ones I could figure out seemed to work fine.
